Ginger/GingerCore/Platforms/PlatformsInfo/PlatformInfoBase.cs

```diff
@@ -35,6 +36,7 @@ public abstract class PlatformInfoBase
         static PowerBuilderPlatform mPowerBuilderPlatform = new PowerBuilderPlatform();
         static WindowsPlatform mWindowsPlatform = new WindowsPlatform();
         internal List<eElementType> mElementsTypeList = null;
+        internal List<ElementTypeOperations> mWebPlatformElementTypeOperations = null;
         internal List<eLocateBy> mElementLocatorsTypeList = null;
```

Comment:
we can't have Web related names like mWebPlatformElementTypeOperations, it needs to be generic, this seems that need to be moved to WebPlatformInfo.cs

Reply:
this is still not in correct place